Model class definition error: Explicit "Any" is not allowed

When I create `model.py` with the following content:


```python
# -*- coding: utf-8 -*-

from django.contrib.auth import get_user_model
from django.db import models

User = get_user_model()


class BlogPost(models.Model):
    """Example model with different fields and a foreign key."""

    author = models.ForeignKey(User, on_delete=models.CASCADE)

    text = models.TextField()

    is_published = models.BooleanField(default=False)
    created_at = models.DateTimeField(auto_now_add=True)

    def __str__(self) -> str:
        """Custom representation."""
        reveal_type(self.id)
        return '<BlogPost {0}>'.format(self.id)
```

And following setting:

```ini
[mypy]
disallow_any_explicit = True

plugins =
  mypy_django_plugin.main

[mypy.plugins.django-stubs]
django_settings_module = server.settings
```

That's what I get:

```
» PYTHONPATH=${PYTHONPATH}:${PWD} mypy server
server/apps/main/models.py:9: error: Explicit "Any" is not allowed
server/apps/main/models.py:21: note: Revealed type is 'builtins.int*'
```

Reproduction link: https://github.com/sobolevn/django_stubs_example (note, that you have to uncomment `disallow_any_explicit = True` in `setup.cfg`)
Versions: django2.2, python3.7, django-stubs1.1

I am not sure why `server/apps/main/models.py:9: error: Explicit "Any" is not allowed` is raised. And I guess, that it should not be. Since there's not explicit `Any`.

--- a/mypy_django_plugin/transformers/models.py
+++ b/mypy_django_plugin/transformers/models.py
@@ -557,8 +557,8 @@ class AddExtraFieldMethods(ModelClassInitializer):
                     name=f"get_next_by_{field.attname}",
                     args=[
                         Argument(
-                            Var("kwargs", AnyType(TypeOfAny.explicit)),
-                            AnyType(TypeOfAny.explicit),
+                            Var("kwargs", AnyType(TypeOfAny.implementation_artifact)),
+                            AnyType(TypeOfAny.implementation_artifact),
                             initializer=None,
                             kind=ARG_STAR2,
                         )
@@ -570,8 +570,8 @@ class AddExtraFieldMethods(ModelClassInitializer):
                     name=f"get_previous_by_{field.attname}",
                     args=[
                         Argument(
-                            Var("kwargs", AnyType(TypeOfAny.explicit)),
-                            AnyType(TypeOfAny.explicit),
+                            Var("kwargs", AnyType(TypeOfAny.implementation_artifact)),
+                            AnyType(TypeOfAny.implementation_artifact),
                             initializer=None,
                             kind=ARG_STAR2,
                         )

--- a/tests/typecheck/models/test_extra_methods.yml
+++ b/tests/typecheck/models/test_extra_methods.yml
@@ -40,6 +40,31 @@
                     date = models.DateField()
                     datetime = models.DateTimeField()
 
+-   case: date_datetime_fields_have_get_next_by_get_previous_by_no_explicit_any
+    mypy_config: |
+        [mypy-myapp.models]
+        disallow_any_generics = true
+        disallow_any_explicit = true
+    main: |
+        from myapp.models import MyUser
+        reveal_type(MyUser().get_next_by_date())  # N: Revealed type is "myapp.models.MyUser"
+        reveal_type(MyUser().get_next_by_datetime())  # N: Revealed type is "myapp.models.MyUser"
+        reveal_type(MyUser().get_previous_by_date())  # N: Revealed type is "myapp.models.MyUser"
+        reveal_type(MyUser().get_previous_by_datetime())  # N: Revealed type is "myapp.models.MyUser"
+
+        # accept arbitrary kwargs
+        MyUser().get_next_by_date(arg1=1, arg2=2)
+    installed_apps:
+        - myapp
+    files:
+        -   path: myapp/__init__.py
+        -   path: myapp/models.py
+            content: |
+                from django.db import models
+                class MyUser(models.Model):
+                    date = models.DateField()
+                    datetime = models.DateTimeField()
+
 -   case: get_next_by_get_previous_by_absent_if_null_true
     main: |
         from myapp.models import MyUser

